templates/fabric8-launcher.yaml

```diff
@@ -26,7 +26,6 @@ parameters:
     should be created (ie. https://hostname[:port] or https://ipaddress[:port]).
     Leave the default value when targetting the same OpenShift/Minishift you are
     running this on
-  value: https://openshift.default.svc.cluster.local
```

Comment:
The template is always copied from the launcher-application repository using the `update-template.sh`
 Can we keep this default value and set it to blank in the operator?

Reply:
changed the apiUrl to be an empty string by default :)